src/hooks/useQuery.ts

```diff
@@ -33,19 +33,34 @@ export type UseQueryResponse<T> = [
 export const useQuery = <T = any, V = object>(
   args: UseQueryArgs<V>
 ): UseQueryResponse<T> => {
+  const isMounted = useRef(false);
```

Comment:
@JoviDeCroock heads up, this is going to conflict with [an existing pr](https://github.com/FormidableLabs/urql/pull/243).

Reply:
Jup, saw that but in theory that invokes a will mount, no?